lib/scoring.rb

```diff
@@ -0,0 +1,90 @@
+module Scrabble
+  class Scoring
+    LETTERS_VALUES = {
```

Comment:
This hash of arrays definitely works, but looking up a letter's score ends up being complicated, involving an `each` loop.

Instead, what if you switched it around and had the keys in this array be letters, and the values be the corresponding score. So it would look like

```ruby
LETTERS_VALUES = {
  'a': 1,
  'b': 3,
  # ...
}
```

Then looking up a letter would be as simple as `LETTERS_VALUES[letter]`.